gpiozero/output_devices.py

```diff
@@ -300,7 +299,10 @@ def __init__(self, pin=None, active_high=True, initial_value=0, frequency=100):
         self._controller = None
         if not 0 <= initial_value <= 1:
             raise OutputDeviceBadValue("initial_value must be between 0 and 1")
-        super(PWMOutputDevice, self).__init__(pin, active_high)
+        if initial_value in (0, 1):
+            super(PWMOutputDevice, self).__init__(pin, active_high, initial_value=initial_value)
```

Comment:
This is a little redundant; when `pin.frequency` is set below that starts the pin implementation's PWM engine (whatever it may be). These all default to a duty cycle of "off" so if the initial-value is 1, the pin'll just be driven back to 0 before we set value on line 309 (hence the comment about needing a method for setting frequency and value together).